ui/components/app/modals/hold-to-reveal-modal/hold-to-reveal-modal.tsx

```diff
@@ -12,11 +12,11 @@ import {
   Box,
   Button,
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Text,
 } from '../../../component-library';
+import { ModalContent } from '../../../component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../component-library/modal-header/deprecated';
 import HoldToRevealButton from '../../hold-to-reveal-button';
 
 type HoldToRevealModalProps = {
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 2 38 40 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/0ec24202-7995-45ec-a387-336dd16c676a">

### After
<img width="1512" alt="Screenshot 2023-12-10 at 2 38 49 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/b7188e87-1e8e-4e56-a9db-2ff8bb0858ab">